src/api/messages/getSingleMessage.js

```diff
@@ -0,0 +1,57 @@
+/* @flow strict-local */
+
+import type { Auth, ApiResponseSuccess } from '../transportTypes';
+import type { Message } from '../apiTypes';
+import { transformFetchedMessage, type FetchedMessage } from '../rawModelTypes';
+import { apiGet } from '../apiFetch';
+import { identityOfAuth } from '../../account/accountMisc';
+
+// The actual response from the server.  We convert the message to a proper
+// Message before returning it to application code.
+type ServerApiResponseSingleMessage = {|
+  ...$Exact<ApiResponseSuccess>,
+  -raw_content: string, // deprecated
+
+  // Until we narrow FetchedMessage into its FL 120+ form, FetchedMessage
+  // will be a bit less precise that we could be here. That's because we
+  // only get this field from servers FL 120+.
+  // TODO(server-5.0): Make this field required, and remove FL-120 comment.
+  +message?: FetchedMessage,
+|};
+
+/**
+ * See https://zulip.com/api/get-message
+ *
+ * Gives undefined if the `message` field is missing, which it will be for
+ *   FL <120.
+ */
+// TODO(server-5.0): Simplify FL-120 condition in jsdoc and implementation.
+export default async (
+  auth: Auth,
+  args: {|
+    +message_id: number,
+    +apply_markdown: boolean,
```

Comment:
We just hardcode this to true in the actual request, right? So we can drop it from the function's parameters.

Reply:
Ah, yeah, right.

Also, I noticed we were accidentally passing `client_gravatar: true`, which this endpoint doesn't accept, as mentioned: https://github.com/zulip/zulip-mobile/pull/5465#discussion_r962044357 so I stopped doing that.